addons/point_of_sale/static/src/js/models.js

```diff
@@ -745,17 +732,9 @@ class PosGlobalState extends PosModel {
         }
 
         var self = this;
-        var timeout = typeof options.timeout === 'number' ? options.timeout : 7500 * server_ids.length;
-
-        return this.env.services.rpc({
-                model: 'pos.order',
-                method: 'remove_from_ui',
-                args: [server_ids],
-                kwargs: {context: this.env.session.user_context},
-            }, {
-                timeout: timeout,
-                shadow: true,
-            })
+
+        return this.env.services.orm.silent
+            .call('pos.order', 'remove_from_ui', [server_ids])
```

Comment:
just putting a note, this one is moved and renamed in the pos_restaurant refactor and is already using a try/catch block